using-timescaledb/data-retention.md

```diff
@@ -147,72 +69,38 @@ SELECT remove_retention_policy('conditions');
+SELECT * FROM timescaledb_information.policy_stats;
 ```
 
 For more information, please see the [API documentation][add_retention_policy].
 
+### Data Retention with Continuous Aggregates
 
-### Using External Job Schedulers
-
-While the built-in scheduling framework automates data retention, the
-`drop_chunks` command can be combined with an external tool for job scheduling,
-like `crontab` or `systemd`, to schedule such commands.
-
-#### Using Crontab
-
-The following cron job will drop chunks every day at 3am:
-
-```bash
-0 3 * * * /usr/bin/psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT drop_chunks('conditions', INTERVAL '24 hours');" >/dev/null 2>&1
-```
-
-The above cron job can easily be installed by running `crontab -e`.
-
-
-#### Using a Systemd Timer
-
-On a systemd-based OS (most modern Linux distributions), a systemd [unit][] with
-accompanying unit [timer][] can also be used to implement a
-retention policy.
-
-First, create, e.g., `/etc/systemd/system/retention.service` unit file:
-
-```ini
-[Unit]
-Description=Drop chunks from the 'conditions' table
-
-[Service]
-Type=oneshot
-ExecStart=/usr/bin/psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT drop_chunks('conditions', INTERVAL '24 hours');"
-```
-
-Then create a timer to run this unit, e.g., `/etc/systemd/system/retention.timer`:
-
-```ini
-[Unit]
-Description=Run data retention at 3am daily
-
-[Timer]
-OnCalendar=*-*-* 03:00:00
-
-[Install]
-WantedBy=timers.target
-```
-
-Once these units have been created, run
+Extra care must be taken when using rention policies or `drop_chunks` calls on
+hypertables which have [continuous aggregates][continuous_aggregates] defined on
+them. Any chunks which are dropped in the region still being refreshed by the
+continuous aggregate will cause the chunk data to disappear from the aggregate as
+well. If the intent is to keep the aggregate while dropping the underlying data,
+the interval being dropped should not overlap with the offsets for the continuous
+aggregate.  More detail on this can be found [here][cagg_drop_data].
+
+One additional item to note is that continuous aggregates are also valid targets
+for `drop_chunks` and retention policies.  As an example, assume you have your
+`conditions` hypertable from above and a `conditions_summary_daily` continuous
+aggregate which stores the average temperature per device per day.  The following
+will drop the data from `conditions` after 30 days, but will keep the
+`conditions_summary_daily` around for an additional 570 days.
 
+```sql
+SELECT add_retention_policy('conditions', INTERVAL '30 days');
+SELECT add_retention_policy('conditions_summary_daily', INTERVAL '600 days');
 ```
-systemctl daemon-reload
-systemctl enable retention.timer
-systemctl start retention.timer
-```
-
-To make systemd load the timer unit, enable it by default on bootup,
-and immediately start it.
 
+Note that for this to work properly, it is important that the `end_offset` for
```

Comment:
```suggestion
For this to work properly, it is important that the `end_offset` for
```

Reply:
Should we show this explicitly in the example, then?  I'm trying to follow this, but with the changes I see proposed, I don't see the definition for conditions_summary_daily anywhere:

![image](https://user-images.githubusercontent.com/205086/94634671-a7e7b280-029e-11eb-83bc-f9ca3398bbb6.png)

Reply:
I've updated the example, hopefully it's much easier to follow now.